suite-native/module-settings/src/components/ColorSchemePickerItem.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { ColorValue, TouchableOpacity } from 'react-native';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { colorVariants, ThemeColorVariant } from '@trezor/theme';
+import { Box, Text } from '@suite-native/atoms';
+
+type ColorSchemePickerItemProps = {
+    isSelected?: boolean;
+    onPress: () => void;
+    colorScheme: ThemeColorVariant;
+};
+
+const pickerItemWrapperStyle = prepareNativeStyle<{ isSelected: boolean }>(
+    (utils, { isSelected }) => ({
+        backgroundColor: utils.colors.gray100,
+        borderRadius: utils.borders.radii.medium,
+        height: 114,
+        flex: 1,
+        paddingTop: 33,
+        borderWidth: utils.borders.widths.medium,
+        borderColor: isSelected ? utils.colors.green : utils.colors.gray100,
+    }),
+);
+
+type PickerItemDotStyleProps = {
+    bgColor: ColorValue;
+    isFirstItem: boolean;
+};
+const pickerItemDotStyle = prepareNativeStyle<PickerItemDotStyleProps>(
+    (utils, { bgColor, isFirstItem }) => ({
+        width: 26,
+        height: 26,
+        backgroundColor: bgColor,
```

Comment:
```suggestion
        backgroundColor,
```
nit: Why not use `backgroundColor` instead of the shortcut and then use it directly?

Reply:
[6f18eaf](https://github.com/trezor/trezor-suite/pull/5693/commits/6f18eaf7956614d05a8f03a0f397ee2a2d8ced05)